src/app/dataset-view/additional-components/metadata-component/components/form-components/schema-field/schema-field.component.html

```diff
@@ -12,12 +12,48 @@
         >
             <ng-container matColumnDef="name">
                 <th mat-header-cell *matHeaderCellDef width="40%">name</th>
-                <td mat-cell *matCellDef="let element" [formGroup]="element">
+                <td
+                    mat-cell
+                    *matCellDef="let element; let i = index"
+                    [formGroup]="element"
+                >
                     <input
                         type="text"
                         placeholder="Enter name"
                         formControlName="name"
                     />
+                    <ng-container
+                        *ngIf="
+                            items.controls[i].get('name')?.hasError('unique')
+                        "
+                    >
+                        <small class="text-danger ps-2">
+                            Name is not unique</small
+                        >
+                    </ng-container>
+                    <ng-container
+                        *ngIf="
+                            items.controls[i].get('name')?.hasError('noneOf')
+                        "
+                    >
+                        <small class="text-danger ps-2">
+                            Type cannot be name</small
```

Comment:
I don't like this approach of error reporting. Have something in TS part that would return you single error message, and have only 1 error block in the HTML part.